pkg/master/master.go

```diff
@@ -226,6 +232,17 @@ type Master struct {
 // NewEtcdStorage returns a StorageInterface for the provided arguments or an error if the version
 // is incorrect.
 func NewEtcdStorage(client tools.EtcdClient, version string, prefix string) (etcdStorage tools.StorageInterface, err error) {
+	if strings.HasPrefix(version, "exp/") {
```

Comment:
It doesn't look hard to make this generic -- just split on `/` and pass in the "latest" API version as a parameter.